src/main/java/homework_2/random_chars_table/RandomCharsTable.java

```diff
@@ -0,0 +1,88 @@
+package homework_2.random_chars_table;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.InputMismatchException;
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+
+public class RandomCharsTable {
+
+    private static void randomCharsTable(int lengthOfTable, int widthOfTable, String strategy) {
```

Comment:
![image](https://user-images.githubusercontent.com/58872509/126910486-47e873a1-664f-4d47-b7fc-1f6623aec1d3.png)
There are, it seems, unnecessary whitespaces, please remove

Reply:
Thank you! Done:)